src/main/kotlin/dev/znci/rocket/scripting/ScriptManager.kt

```diff
@@ -41,6 +38,7 @@ object ScriptManager {
             globals.set("events", LuaEvents())
             globals.set("commands", LuaCommands())
             globals.set("http", LuaHTTPClient())
+            globals.set("location", LuaLocations())
```

Comment:
This is fine for now, however in the future we will hopefully add some sort of API for this thing, since we want to have an add-on API.